crates/oxc_language_server/src/linter/tester.rs

```diff
@@ -0,0 +1,131 @@
+use oxc_linter::Linter;
+use tower_lsp::lsp_types::Url;
+
+use super::{error_with_position::DiagnosticReport, server_linter::ServerLinter};
+
+const CURRENT_DIR: &str = env!("CARGO_MANIFEST_DIR");
+
+/// Given a file path relative to the crate root directory, return the URI of the file.
+pub fn get_file_uri(relative_file_path: &str) -> Url {
+    let absolute_file_path = PathBuf::from(CURRENT_DIR).join(relative_file_path);
+    let file_path = format!("file://{}", absolute_file_path.display());
+    Url::parse(&file_path).unwrap()
```

Comment:
is it possible to remove the `file://` prefix and use `Url::from_file_path` instead?
Did not test my suggestion

Reply:
Good idea, I didn't look too closely at what other `Url` methods were available, so I didn't know about this one, thanks for the suggestion.